runtime/src/cost_model.rs

```diff
@@ -153,19 +160,25 @@ impl CostModel {
         data_bytes_cost
     }
 
-    fn get_transaction_cost(&self, transaction: &SanitizedTransaction) -> u64 {
-        let mut cost: u64 = 0;
+    fn get_transaction_cost(&self, transaction: &SanitizedTransaction) -> (u64, u64) {
+        let mut builtin_costs = 0u64;
+        let mut bpf_costs = 0u64;
 
         for (program_id, instruction) in transaction.message().program_instructions_iter() {
-            let instruction_cost = self.find_instruction_cost(program_id);
-            trace!(
-                "instruction {:?} has cost of {}",
-                instruction,
-                instruction_cost
-            );
-            cost = cost.saturating_add(instruction_cost);
+            // to keep the same behavior, look for builtin first
```

Comment:
same as what? i think this comment assumes the reader has context that they may not. or worse encodes a property of this PR which has no future value